packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -221,6 +222,25 @@ const EDiscovery = SparkPlugin.extend({
       .then((res) => res.body);
   },
 
+  @waitForValue('@')
+  /**
+   * Retrieves user information from CI based on a query filter
+   * @param {UUID} orgId - Id of the organisation for which users are being requested
+   * @param {String} filter - Query containing information to match users on (id, username, displayName etc.)
+   * @returns {Promise<ResponseEntity<Resources>>} Http response containing the user information
+   */
+  getUsers(orgId, filter = '') {
+    if (!orgId) {
+      throw Error('Undefined parameter');
```

Comment:
Instead of a generic parameter message, would this be more helpful as "organization ID needed to get users"?